nano/node/active_transactions.cpp

```diff
@@ -806,6 +812,11 @@ nano::election_insertion_result nano::active_transactions::insert_impl (nano::un
 				election_behavior_a);
 				roots.get<tag_root> ().emplace (nano::active_transactions::conflict_info{ root, result.election, epoch, election_behavior_a });
 				blocks.emplace (hash, result.election);
+				// Increase hinted election counter while still holding lock
```

Comment:
Is this safe? What if the election already exists?

Reply:
This should be safe because of the check `existing == roots.get<tag_root> ().end ()` further up.